style.css

```diff
@@ -100,6 +107,367 @@ ul li {
+  overflow: hidden;
+}
+
+.my-main-works .single-project .project-image .single-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.my-main-works .single-project .project-image .single-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.my-main-works .single-project .project-image .single-language {
+  display: flex;
+  gap: 12px;
+  flex-direction: row;
+}
+
+.my-main-works .single-project .project-image .single-language li {
+  padding: 10px 12px;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  text-align: center;
+}
+
+.single-project .view-project {
+  width: 100%;
+  height: 48px;
+  background-color: #ff6b00;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.single-project .view-project #view-project-id {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+/* single project section end */
+
+/* about me section start */
+.about-me {
+  width: 100%;
+  height: 1901px;
+  background-image: url('./images/about-me.png');
+  background-repeat: no-repeat;
+  background-position: right center;
+  background-position-y: 350.6px;
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 120px;
+  gap: 12px;
+}
+
+.about-me .container {
+  /* background-image: url('./images/about-me.png');
+  background-repeat: no-repeat;
+  background-position: right center;
+  background-position-y: 432.6px; */
+  display: flex;
+  justify-content: flex-start;
+  min-width: 100%;
+}
+
+.about-me h3 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  width: 327px;
+  height: 79px;
+}
+
+.about-me #about-me-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  width: 327px;
+  height: 148px;
+  color: #f4f5f7;
+}
+
+.about-me button {
+  width: 161px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+  text-align: center;
+  letter-spacing: 0.03em;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.about-me button:hover {
+  color: #ff6b00;
+  background-color: #fff;
+  border: 0.5px solid #ff6b00;
+}
+
+.about-me button:focus {
+  background: #ff6b00;
+  color: #fff;
+}
+
+.about-me hr {
+  width: 100%;
+  height: 0;
+  margin-top: 201px;
+  margin-right: 16px;
+}
```

Comment:
- [ ] Kindly use more gaps between the `hr` and `about-my-self` to match the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=3CbGHNhPHWBiQT4V-0).
  - ***Left one is your design, and the Right one is Figma design***
  ![Screenshot from 2023-01-22 00-26-07](https://user-images.githubusercontent.com/60083437/213881713-70a1244c-9fa9-4948-9530-361ab52cec39.png)